tasks/test-project/test-project

```diff
@@ -160,7 +160,7 @@ const copyProject = async () => {
     // copy from tempDir to Fixture dir
     await fse.copy(OUTPUT_PROJECT_PATH, FIXTURE_TESTPROJ_PATH)
     // cleanup after ourselves
-    rimraf.sync(OUTPUT_PROJECT_PATH)
+    // rimraf.sync(OUTPUT_PROJECT_PATH)
```

Comment:
Are you doing this temporarily in order to inspect and compare the output prior to copying to fixtures/test-project?

Reply:
Yes I was for testing..

How do you feel about leaving the output in the temp directory? I think its useful to have it there when we debug and since its the temp directory, no harm done (just like e2e)